src/node_nanomsg.cc

```diff
@@ -23,6 +23,11 @@ using v8::Object;
 using v8::String;
 using v8::Value;
 
+#define ret NanReturnValue
+#define utf8 v8::String::Utf8Value
```

Comment:
ya but i'm talking about the unsigned int thing.

when declaring ints, i was using `As()->IntegerValue()` on `args[0]` as opposed to:

```c++
int s = args[0]->Uint32Value();
```